web/views/layout.erb

```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```

Comment:
If someone wants to send a PR changing the poll_js logic in the same manner, I'd love it.